packages/block-library/src/template-part/index.php

```diff
@@ -70,7 +70,9 @@ function render_block_core_template_part( $attributes ) {
 			if ( 0 === validate_file( $attributes['slug'] ) ) {
 				$block_template = get_block_file_template( $template_part_id, 'wp_template_part' );
 
-				$content = $block_template->content;
+				if ( isset( $block_template->content ) ) {
```

Comment:
Looking at the code after the check, it feels to me that `$content` should default to an empty string so it goes through the filter `render_block_core_template_part_none` covering the case when the template doesn't exist. The existing check is very restrictive:

```php
if ( '' !== $content && null !== $content ) {
```